examples/client-logger/app-insights-logger/.eslintrc.js

```diff
@@ -0,0 +1,11 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+module.exports = {
+	extends: [require.resolve("@fluidframework/eslint-config-fluid/minimal"), "prettier"],
```

Comment:
We shouldn't be using the minimal config in any new packages. `strict` is always my recommendation, though `recommended` is fine for test apps.